styles.css

```diff
@@ -110,12 +117,314 @@ section {
 }
 
 .second-section p:nth-child(3) {
+  color: #344563;
   margin-top: 12px;
-  width: 90%;
 }
 
 .social-container li {
-  margin-right: 19px;
-  font-size: 18px;
+  margin-right: 20px;
+  font-size: 25px;
+  width: 20px;
+  color: #505f79;
+}
+
+.projects {
+  display: grid;
+  grid-template-columns: 100%;
+  grid-column-gap: 50px;
+  width: 100%;
+  margin: auto;
+}
+
```

Comment:
- [ ] Nice work to see the main container of the projects has a display:grid:heavy_check_mark:

Reply:
Thanks, Emmy.

Reply:
No issue